data-azure-cosmos/src/main/java/io/micronaut/data/cosmos/operations/CosmosBinder.java

```diff
@@ -138,10 +138,13 @@ private ConversionContext createTypeConversionContext(@Nullable RuntimePersisten
         return ConversionContext.DEFAULT;
     }
 
+    @SuppressWarnings("java:S3824") // Disabled Sonar Rule: "Map.get" and value test should be replaced with single method call
     private void doBind(@NonNull QueryParameterBinding binding, Object value, String parameterName) {
         if (updateQuery) {
             String property = getUpdateProperty(binding, persistentEntity);
```

Comment:
This is what I was worried:) Sonar suggested use computeIfAbsent vs containsKey but the thing is we need to put null values (update date = null where x = y) and computeIfAbsent wouldn't work with it then. So this is quick bugfix for the issue introduced by Sonar fixes